sdk/appplatform/Microsoft.Azure.Management.AppPlatform/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Azure Spring Cloud Management Library")]
+[assembly: AssemblyDescription("Provides developers with libraries for the Azure Spring Cloud under Azure Resource manager to deploy, start, stop and scale Spring Cloud applications.")]
+
+[assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
Use `0.1.0.0` for `AssemblyVersion` and `AssemblyVersion` please.
After your module goes stable, update `AssemblyVersion` only when there's a major version change, and update `AssemblyFileVersion` every time.

Reply:
@jialindai please fix the versions. They matter because they are used by the CLR when loading your DLL. See

Reply:
@isra-fel , I updated the versions. Please take a look.